### DISCUSSION TITLE
it works, but not reachs the target FPS

### DISCUSSION THREAD

**Discussion by MilesCSC** [34736384/genshin-fps-unlock#324] [Q&A]
It only runs on the max fps of my monitor(144hz) although i set 420 as the target FPS.
Can i get higher? I want to try sth that need as high FPS as possible
Set up 5900X 3060ti MSI afterburner as FPS monitor
![螢幕擷取畫面 2023-12-27 215054](https://github.com/34736384/genshin-fps-unlock/assets/99173499/8b7d1a24-f275-45ef-8a08-399cdb396736)

**Comment by MenonPlayz** [comment]
Do you have Vsync enabled somewhere?

**Reply by MilesCSC** [reply]
SOLVED!! thanks for reply, it's cause by Vertical Synchronization, just like you say ( in game setting).
